src/PayID/pay-id-client.ts

```diff
@@ -22,7 +24,53 @@ export default class PayIDClient {
    * @param payID The payID to resolve for an address.
    * @returns An XRP address representing the given PayID if one exists, otherwise undefined.
    */
-  public async xrpAddressForPayID(_payID: string): Promise<string | undefined> {
-    throw PayIDError.unimplemented
+  public async xrpAddressForPayID(payID: string): Promise<string | undefined> {
+    const paymentPointer = PayIDUtils.parsePaymentPointer(payID)
+    if (!paymentPointer) {
+      throw PayIDError.invalidPaymentPointer
+    }
+
+    const url = `https://${paymentPointer.host}${paymentPointer.path}`
+    // TODO(keefertaylor): Attach network parameter.
+    // TODO(keefertaylor): Generalize the below to allow requesting in other types.
+    const requestConfig = {
+      headers: {
+        Accept: 'application/xrp+json',
```

Comment:
Fine for now, but I believe we're going to update this to reflect the formats in this doc: https://docs.google.com/document/d/1_Jl12pWqfVGmpq5hhOXClOJXuMck0dEaF_NvgvYkWSg/edit like 'xrp-mainnet' and 'xrp-testnet'